src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -0,0 +1,414 @@
+  @FindBy(css = "li.private-allow")
+  private WebElement allowPrivateMassageButton;
+  @FindBy(css = "li.kick")
+  private WebElement kickUserButton;
+  @FindBy(css = "li.private-block")
+  private WebElement blockPrivateMassageButton;
+  @FindBy(css = "h1.public.wordmark.selected")
+  private WebElement mainChatSelection;
+  @FindBy(css = "ul.PrivateChatList span.splotch")
+  private WebElement privateMessageNotification;
+  @FindBy(css = ".continued.inline-alert")
+  private WebElement chatInlineAlertContinued;
+  @FindBy(css = "#UserStatsMenu li.ban")
+  private WebElement banUserButton;
+  @FindBy(css = "#ChatBanModal")
+  private WebElement chatBanModal;
+  @FindBy(css = "#ChatBanModal button.primary")
+  private WebElement chatBanModalButton;
+  @FindBy(css = "#UserStatsMenu .regular-actions li")
+  private List<WebElement> userDropDownActionsElements;
+  @FindBy(css = "#UserStatsMenu .admin-actions li")
+  private List<WebElement> adminDropDownActionsElements;
+  @FindBy(css = "#Rail h1.private")
+  private WebElement privateMessagesHeader;
+  @FindBy(css = "#Rail h1.wordmark")
+  private WebElement chatWordmarkImage;
+  @FindBy(css = "#WikiChatList li")
+  private WebElement chatLoadedIndicator;
+  @FindBy(css = "#ChatHeader h1.private")
+  private WebElement privateChatHeader;
+  @FindBy(css = "div.actions .admin-actions .give-chat-mod")
+  private WebElement addUserModStatusButton;
+  @FindBy(css = "div#UserStatsMenu.UserStatsMenu")
+  private WebElement userOptions;
+  @FindBy(css = "ul.admin-actions")
+  private WebElement userModOptions;
+  @FindBy(css = "li.message-wall")
+  private WebElement userMessageWallButton;
+  @FindBy(xpath = "//li[@data-id='wall']")
+  private WebElement userPageMessageWallTab;
+  @FindBy(css = "li.contribs")
+  private WebElement userContributionsButton;
+  @FindBy(xpath = "//li[@data-id='contribs']")
+  private WebElement userPageContributionsTab;
+  @FindBy(css = "form#Write.Write.blocked")
+  private WebElement userBlockedMessageField;
+  @FindBy(css = "div.header-column.header-title")
+  private WebElement permissionsErrorTitle;
+  @FindBy(xpath = "//h1")
+  private WebElement userNameTitle;
+
+  private static final String USER_UNBAN_LINK = "//a[@data-type='ban-undo' and @data-user='%s']";
+  private static final String USER_UNBAN_CONFIRM_MESSAGE =
+          "//div[@class='Chat']//li[contains(text(), 'has ended the Chat ban for %s')]";
+  private static final String USER_SELECTOR = "#user-%s";
+  private static final String PRIVATE_MESSAGE_USER_SELECTOR = "#priv-user-%s";
+  private static final String PRIVATE_MESSAGE_SELECTED_USER_SELECTOR =
+          "#priv-user-%s.User.selected";
+  private static final String PATH_MESSAGE_ON_CHAT = "//span[@class='message'][contains(text(), '%s')]";
+  private static final String NOTIFICATION_COUNTER =
+          "//span[@class='splotch' and contains(text(), '%s')]";
+  private final String USER_NAME_ON_PROFILE_SCREEN = userNameTitle.getText();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "USER_NAME_ON_PROFILE_SCREEN" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)